TRANSLATING.md

```diff
@@ -91,50 +98,40 @@ If you spot something on [docs.astro.build](https://docs.astro.build/) that you
 
     ➤ Go to `src/pages/{language}/{page-slug}.md`
 
-## Contributing to translations
+# Contributing to translations
 
 Please see [CONTRIBUTING.md](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) for information about contributing via a fork, our Style Guide, and more!
----
 
-## Adding a new language
+## Review Tips
 
-> 🛑 **Please don’t add a new language without first consulting with the docs team in [the `#docs-i18n` channel on Discord](https://astro.build/chat).**
+Reviewing PRs is an important task — it's thanks to them that we can guarantee consistent, high-quality translations, so if you're interested in helping review translation PRs, thank you, we really appreciate the effort!
 
-### Prerequisites 
+**When you think a PR is good to be merged**, approve the PR through GitHub's "Review Changes" button or leave a "**LGTM!**" in the comments. (“LGTM” is an abbreviation of “Looks Good to Me” often used to approve pull requests.)
```

Comment:
I think I would put this after the checklist, since approving it happens AFTER they've gone through the checklist!

Reply:
Fair point! Just applied this change